.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
delete this file from the branch and perhaps add `.vscode/` to the `.gitignore` file instead?

Reply:
Changed